procrastinate/admin.py

```diff
@@ -0,0 +1,181 @@
+from typing import Any, Dict, Iterable
+
+from procrastinate import connector as connector_module
+from procrastinate import sql, utils
+
+
+@utils.add_sync_api
+class Admin:
+    """
+    The Admin is used to overview and administrate procrastinate jobs.
+
+    You should never need to instanciate an Admin object as it is
+    already available as `App.admin`.
+    """
+
+    def __init__(self, connector: connector_module.BaseConnector):
+        """
+        Parameters
+        ----------
+        connector:
+            Instance of a subclass of :py:class:`procrastinate.connector.BaseConnector`,
+            typically `PostgresConnector`. It will be responsible for all communications
+            with the database. Mandatory.
+        """
+        self.connector = connector
+
+    async def list_jobs_async(
+        self,
+        id: int = None,
+        queue: str = None,
+        task: str = None,
+        status: str = None,
+        lock: str = None,
+    ) -> Iterable[Dict[str, Any]]:
+        """
+        List all procrastinate jobs given query filters.
+
+        Parameters
+        ----------
+        id: int
```

Comment:
We use spaces around `:` elsewhere in the code, and double-backticks around native types. So
```
id : ``int``
```
Not sure how important it is for the doc generation. @ewjoachim?

Reply:
Not everywhere: https://github.com/peopledoc/procrastinate/blob/64472ff6559dffaffb3d39f53217c1c2a99e60b7/procrastinate/app.py#L184-L189

But I'm ok to add spaces and backticks.

Reply:
Yep, not that consistent indeed.

Reply:
I'd say: don't bother with that yet. If we want to enforce that someday, we'll do it with linters. Meanwhile, as long as sphinx & napoleon are happy, that's enough to me.